### ISSUE TITLE
[Documentation] Formalise the blog publish process

### ISSUE THREAD

**Issue by Emily-Jiang** [microprofile/microprofile-marketing#484]
Recently I wrote  a blog and tried to follow this blog publishing [guide](https://github.com/microprofile/wpsite). However, it is not a process. Because of this, we have different understanding on when and how to publish a blog.

We should define a process:
step 1: submit a blog
step 2: raise a github issue
step 3: the editor team to review within x days
step 4: address feedback and publish

We need to have a clear plan on this.... low bar entry and quick turnabout. We need to use a tag `blog` and marketing calls need to look at these issues and address them weekly.

**Comment by aeiras** [comment]
Expectations setting is great. 

Usually, if I blog arrives, it should get a reply within 3 days, max ( week days)
Your ticket blog, I didn't see it arrive. Yet I saw its closing notification that triggered my review :) 
So you see, We need to enable the reviewer to be welcomed to pull Editors to the ticket if 3 days pass. All within the Active Reviewers (only a few of us currently help) Easy. 

Keeping the bar low and easy to own is very important. I always say, lets try stuff and adjust. Formalizing the review process is good for us. I am very happy we are able to do this.

**Comment by TetianaFTV** [comment]
Hi, 

I've prepared a draft with instructions on how to submit an article on MicroProfile blog
https://docs.google.com/document/d/1BwBLs88AhkOmxF8KEuTMosyPaOxOdMsUm7VJrHmIrrE/edit#

Please review and edit as required.

On the call there was also mentioned option to create an issue template that has a checklist:
- Review of Google Docs draft (optional)
- Review of uploaded article
- Publish article
- Post the Tweet

If this is going to be done, then we can add in the instructions that the git issue should be created based on this specific template. Are we going to create it? If yes, who is going to do that?

**Comment by aeiras** [comment]
@Emily-Jiang & @TetianaFTV, 

I have reviewed the document drafted, linked it to current admins document (moved to INFRA folder) & the regular guidelines. 
I also checked the 2021 active wordpress Admin/Editors as well. 
A major part is missing, I pulled you via comments Tetiana as content contribution always have 2 parts, "need for editors to review content" & no need to copy/edit. However, even if the google document is not dropped or requested, the Reviewer might recommend the conversation of the WordPress blog to google document to enable that "must" review. That needs to be let to the sole discression of the MicroProfilers attending the review. 
Also, This document serves to the Editor/Admins who help with publishing. So it needs to speak to them as well. 365. It is important for the Reviewer to feel welcomed to submit a marketing forum email asking for a follow up on any review ticket, ideally the conversation lands into the Mktg call & is discussed.   
The document misses the your article won't be published in the MP blog + potential reasons. This happens so it needs to be stated in the document as well. 

Really good start,
